src/utils/getTestFrameworkName.ts

```diff
@@ -1,4 +1,11 @@
+// Import Node.js Dependencies
+import fs from "fs";
+import path from "path";
```

Comment:
```suggestion
import fs from "node:fs";
import path from "node:path";
```
(This should even be removed as we will not parse package.json again)